src/components/domain/Summary.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { FunctionComponent } from "react";
```

Comment:
Could the exports here be extracted to their own files, so each file has 1x exported React component?  I think this makes the components more discoverable, making it easier to discover and re-use components.  This might necessitate a bit of renaming of some of the components so they make more sense in isolation.

We don't explicitly follow it, but the AirBnB style guide suggests this too: https://github.com/airbnb/javascript/blob/master/react/README.md#basic-rules